gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -48,7 +48,6 @@ def add_arguments(self, parser):
   def validate(self, parsed_args):
     # type: (argparse.Namespace) -> None
     """Validates this group's options parsed from the command line."""
-    pass
```

Comment:
I'd like to make sure removing this does not affect the flag validation behavior. Could you please confirm this change is intentional?

Reply:
Yeah it is required, by pylint. python 3 doesn't require pass anymore.